osidb/models.py

```diff
@@ -796,6 +796,36 @@ def _validate_cwe_format(self):
             if not re.match(r"^(CWE-[0-9]+|\(CWE-[0-9]+(\|CWE-[0-9]+)*\))$", element):
                 raise ValidationError("CWE IDs is not well formated.")
 
+    def _validate_unsupported_impact_change(self):
+        """
+        Check that an update of a flaw with open trackers does not change between
+        Critical/Important/Major Incident and Moderate/Low.
+        """
+        if self._state.adding:
+            return
+
+        old_flaw = Flaw.objects.get(pk=self.pk)
+        was_high_impact = (
+            old_flaw.impact in [FlawImpact.CRITICAL, FlawImpact.IMPORTANT]
+            or old_flaw.is_major_incident
+        )
+        still_high_impact = (
+            self.impact in [FlawImpact.CRITICAL, FlawImpact.IMPORTANT]
+            or self.is_major_incident
+        )
+        if (
+            was_high_impact
+            and not still_high_impact
+            and Tracker.objects.filter(affects__flaw=self)
+            .exclude(status="Closed")
+            .exists()
+        ):
```

Comment:
I am checking the SFM2 validation and there is an important difference

`A change of impact from Critical/Important/Major Incident to Moderate/Low **and vice versa** is not supported by the tooling.`

where this validation does just one half (one direction). Do you plan to create another validation or does this one need to be adjusted? Basically instead of **and not** you might have **!=** but also the error text below would have to be adjusted accordingly so it applies also for eg. low-to-important change.

Reply:
makes sense, submitting fixes.